Make seamless-login service less aggressive to allow successful restoration of sessions

ituri opened the pull request:
Use `Restart=on-failure` and increase `RestartSec=5`  to fix resume race condition.

Closes #412

ryanrhughes commented:
I'm running into some issues here that I think we need to address before merging something like this in. 

This change kills the current method of relaunching Hyprland from the power menu `uwsm stop`, as well as the prior method `hyprctl dispatch exit` because they're both considered clean exits. We would need to find a new way to accomplish this, as well as understand that if any of those exit now, you're rebooting.

I'm also curious if you tried just extending the RestartSec alone vs messing with the restart condition here? If the issue you're encountering is one of a race condition, that should really be sufficient.

ituri commented:
You've got a point there. I have just gone back to setting back `Restart=always` while sticking to the `RestartSec=5` setting. This seems to work fine as well.

Any thoughts on just increasing the `RestartSec` then and incorporating that into future releases?

ryanrhughes commented:
We could do that but I also worry that we may just be masking an underlying problem. There was [a video posted by a user in Discord](https://discordapp.com/channels/1390012484194275541/1390012487000395859/1400662562453979229) (maybe also you?) that cited a similar issue.

In that video, I was able to capture that the error is that there's already a graphical state active so it's as though their system is trying to re-launch on top of itself. 

<img width="2836" height="1592" alt="Image" src="https://github.com/user-attachments/assets/2fee38f8-f6d3-46a9-a958-f3fbc226132e" />

Based on the logs you posted in #412, I can see some things happening that I don't expect to but it's unclear to me if it's during the suspend action or when the lid is reopened since the timing is pretty continuous.

Could I convince you to restore the restart to where it causes the failure for you then perform a test making note of the time you close the lid and the time you open the lid so we can compare what's happening in the logs to find the core cause? I've included my logs in the collapsed section below from my laptop where we can see a number of differences between what I have and what's logged in #412; namely no mentions of seamless-login-service, ss-login, PAM, etc. since it should just be restoring an active session and presenting Hyprlock. All of those items are already running in the frozen / restored memory.

`journalctl --reverse` is what I use to grab logs so it's easier to find the latest set vs scrolling to the bottom if that's helpful at all.

<details>

<summary><h3>Example logs</h3></summary>
Lid closed at 21:12, waited and reopened at 21:15.

```
Aug 01 21:15:57 framework systemd-resolved[1547]: Clock change detected. Flushing caches.
Aug 01 21:15:57 framework systemd-resolved[1547]: Clock change detected. Flushing caches.
Aug 01 21:15:57 framework systemd-timesyncd[1548]: Contacted time server 23.186.168.127:123 (2.arch.pool.ntp.org).
Aug 01 21:15:55 framework systemd-timesyncd[1548]: Network configuration changed, trying to establish connection.
Aug 01 21:15:55 framework systemd-networkd[1572]: wlan0: DHCPv4 address 172.25.2.59/20, gateway 172.25.0.1 acquired from 172.25.0.1
Aug 01 21:15:55 framework systemd-networkd[1572]: wlan0: Gained carrier
Aug 01 21:15:55 framework iwd[1747]: event: state, old: connecting (auto), new: connected
Aug 01 21:15:55 framework systemd-networkd[1572]: wlan0: Connected WiFi access point: Veneto Las Olas_Residents (e6:55:b8:57:36:b1)
Aug 01 21:15:55 framework kernel: wlan0: Limiting TX power to 30 (30 - 0) dBm as advertised by e6:55:b8:57:36:b1
Aug 01 21:15:55 framework kernel: wlan0: associated
Aug 01 21:15:55 framework kernel: wlan0: RX AssocResp from e6:55:b8:57:36:b1 (capab=0x1511 status=0 aid=5)
Aug 01 21:15:55 framework kernel: wlan0: associate with e6:55:b8:57:36:b1 (try 1/3)
Aug 01 21:15:55 framework kernel: wlan0: authenticated
Aug 01 21:15:55 framework kernel: wlan0: send auth to e6:55:b8:57:36:b1 (try 1/3)
Aug 01 21:15:55 framework kernel: wlan0: authenticate with e6:55:b8:57:36:b1 (local address=d8:b3:2f:bd:c7:0d)
Aug 01 21:15:55 framework iwd[1747]: event: state, old: autoconnect_quick, new: connecting (auto)
Aug 01 21:15:55 framework iwd[1747]: event: connect-info, ssid: Veneto Las Olas_Residents, bss: e6:55:b8:57:36:b1, signal: -67, load: 12/255
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for da:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for f6:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for ea:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for ee:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for e2:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for e6:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for 96:76:3f:20:e6:23
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for 8e:76:3f:20:e6:23
Aug 01 21:15:55 framework kernel: usb 1-1: reset full-speed USB device number 2 using xhci_hcd
Aug 01 21:15:54 framework systemd[1]: Started Fingerprint Authentication Daemon.
Aug 01 21:15:54 framework systemd-resolved[1547]: Switching to fallback DNS server 1.1.1.1#cloudflare-dns.com.
Aug 01 21:15:54 framework kernel: usb 1-1: reset full-speed USB device number 2 using xhci_hcd
Aug 01 21:15:54 framework systemd-networkd[1572]: wlan0: Reconfiguring with /etc/systemd/network/20-wlan.network.
Aug 01 21:15:54 framework systemd-timesyncd[1548]: No network connectivity, watching for changes.
Aug 01 21:15:54 framework systemd-resolved[1547]: Resetting learnt feature levels on all servers.
Aug 01 21:15:54 framework systemd-resolved[1547]: Closing all remaining TCP connections.
Aug 01 21:15:54 framework systemd-resolved[1547]: Clock change detected. Flushing caches.
Aug 01 21:15:54 framework systemd[1]: Starting Fingerprint Authentication Daemon...
Aug 01 21:15:54 framework systemd[1]: Stopped target Suspend.
Aug 01 21:15:54 framework systemd-logind[1748]: Operation 'suspend' finished.
Aug 01 21:15:54 framework systemd[1]: Reached target Suspend.
Aug 01 21:15:54 framework systemd[1]: Stopped target Sleep.
Aug 01 21:15:54 framework systemd[1]: Finished System Suspend.
Aug 01 21:15:54 framework systemd[1]: systemd-suspend.service: Deactivated successfully.
Aug 01 21:15:54 framework systemd[1]: user@1000.service: Unit now thawed.
Aug 01 21:15:54 framework systemd-sleep[33661]: Successfully thawed unit 'user.slice'.
Aug 01 21:15:54 framework systemd[1]: session-1.scope: Unit now thawed.
Aug 01 21:15:54 framework systemd[1]: user-1000.slice: Unit now thawed.
Aug 01 21:15:54 framework systemd[1]: user.slice: Unit now thawed.
Aug 01 21:15:54 framework kernel: PM: suspend exit
Aug 01 21:15:54 framework kanata[1916]: 21:15:54.6140 [ERROR] failed poll: Os { code: 4, kind: Interrupted, message: "Interrupted system call" }
Aug 01 21:15:54 framework bluetoothd[1746]: Controller resume with wake event 0x0
Aug 01 21:15:54 framework systemd-sleep[33661]: System returned from sleep operation 'suspend'.
Aug 01 21:15:54 framework systemd-networkd[1572]: wlan0: DHCP lease lost
Aug 01 21:15:54 framework systemd-networkd[1572]: wlan0: Lost carrier
Aug 01 21:15:54 framework iwd[1747]: event: state, old: disconnected, new: autoconnect_quick
Aug 01 21:15:54 framework iwd[1747]: event: state, old: connected, new: disconnected
Aug 01 21:15:54 framework iwd[1747]: event: disconnect-info, reason: 3
Aug 01 21:15:54 framework iwd[1747]: Received Deauthentication event, reason: 3, from_ap: false
Aug 01 21:15:54 framework systemd-logind[1748]: Lid opened.
Aug 01 21:15:54 framework kernel: random: crng reseeded on system resumption
Aug 01 21:15:54 framework kernel: Restarting tasks ... done.
Aug 01 21:15:57 framework systemd-resolved[1547]: Clock change detected. Flushing caches.
Aug 01 21:15:57 framework systemd-timesyncd[1548]: Contacted time server 23.186.168.127:123 (2.arch.pool.ntp.org).
Aug 01 21:15:55 framework systemd-timesyncd[1548]: Network configuration changed, trying to establish connection.
Aug 01 21:15:55 framework systemd-networkd[1572]: wlan0: DHCPv4 address 172.25.2.59/20, gateway 172.25.0.1 acquired from 172.25.0.1
Aug 01 21:15:55 framework systemd-networkd[1572]: wlan0: Gained carrier
Aug 01 21:15:55 framework iwd[1747]: event: state, old: connecting (auto), new: connected
Aug 01 21:15:55 framework systemd-networkd[1572]: wlan0: Connected WiFi access point: Veneto Las Olas_Residents (e6:55:b8:57:36:b1)
Aug 01 21:15:55 framework kernel: wlan0: Limiting TX power to 30 (30 - 0) dBm as advertised by e6:55:b8:57:36:b1
Aug 01 21:15:55 framework kernel: wlan0: associated
Aug 01 21:15:55 framework kernel: wlan0: RX AssocResp from e6:55:b8:57:36:b1 (capab=0x1511 status=0 aid=5)
Aug 01 21:15:55 framework kernel: wlan0: associate with e6:55:b8:57:36:b1 (try 1/3)
Aug 01 21:15:55 framework kernel: wlan0: authenticated
Aug 01 21:15:55 framework kernel: wlan0: send auth to e6:55:b8:57:36:b1 (try 1/3)
Aug 01 21:15:55 framework kernel: wlan0: authenticate with e6:55:b8:57:36:b1 (local address=d8:b3:2f:bd:c7:0d)
Aug 01 21:15:55 framework iwd[1747]: event: state, old: autoconnect_quick, new: connecting (auto)
Aug 01 21:15:55 framework iwd[1747]: event: connect-info, ssid: Veneto Las Olas_Residents, bss: e6:55:b8:57:36:b1, signal: -67, load: 12/255
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for da:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for f6:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for ea:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for ee:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for e2:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for e6:55:b8:57:36:b1
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for 96:76:3f:20:e6:23
Aug 01 21:15:55 framework iwd[1747]: invalid HE capabilities for 8e:76:3f:20:e6:23
Aug 01 21:15:55 framework kernel: usb 1-1: reset full-speed USB device number 2 using xhci_hcd
Aug 01 21:15:54 framework systemd[1]: Started Fingerprint Authentication Daemon.
Aug 01 21:15:54 framework systemd-resolved[1547]: Switching to fallback DNS server 1.1.1.1#cloudflare-dns.com.
Aug 01 21:15:54 framework kernel: usb 1-1: reset full-speed USB device number 2 using xhci_hcd
Aug 01 21:15:54 framework systemd-networkd[1572]: wlan0: Reconfiguring with /etc/systemd/network/20-wlan.network.
Aug 01 21:15:54 framework systemd-timesyncd[1548]: No network connectivity, watching for changes.
Aug 01 21:15:54 framework systemd-resolved[1547]: Resetting learnt feature levels on all servers.
Aug 01 21:15:54 framework systemd-resolved[1547]: Closing all remaining TCP connections.
Aug 01 21:15:54 framework systemd-resolved[1547]: Clock change detected. Flushing caches.
Aug 01 21:15:54 framework systemd[1]: Starting Fingerprint Authentication Daemon...
Aug 01 21:15:54 framework systemd[1]: Stopped target Suspend.
Aug 01 21:15:54 framework systemd-logind[1748]: Operation 'suspend' finished.
Aug 01 21:15:54 framework systemd[1]: Reached target Suspend.
Aug 01 21:15:54 framework systemd[1]: Stopped target Sleep.
Aug 01 21:15:54 framework systemd[1]: Finished System Suspend.
Aug 01 21:15:54 framework systemd[1]: systemd-suspend.service: Deactivated successfully.
Aug 01 21:15:54 framework systemd[1]: user@1000.service: Unit now thawed.
Aug 01 21:15:54 framework systemd-sleep[33661]: Successfully thawed unit 'user.slice'.
Aug 01 21:15:54 framework systemd[1]: session-1.scope: Unit now thawed.
Aug 01 21:15:54 framework systemd[1]: user-1000.slice: Unit now thawed.
Aug 01 21:15:54 framework systemd[1]: user.slice: Unit now thawed.
Aug 01 21:15:54 framework kernel: PM: suspend exit
Aug 01 21:15:54 framework kanata[1916]: 21:15:54.6140 [ERROR] failed poll: Os { code: 4, kind: Interrupted, message: "Interrupted system call" }
Aug 01 21:15:54 framework bluetoothd[1746]: Controller resume with wake event 0x0
Aug 01 21:15:54 framework systemd-sleep[33661]: System returned from sleep operation 'suspend'.
Aug 01 21:15:54 framework systemd-networkd[1572]: wlan0: DHCP lease lost
Aug 01 21:15:54 framework systemd-networkd[1572]: wlan0: Lost carrier
Aug 01 21:15:54 framework iwd[1747]: event: state, old: disconnected, new: autoconnect_quick
Aug 01 21:15:54 framework iwd[1747]: event: state, old: connected, new: disconnected
Aug 01 21:15:54 framework iwd[1747]: event: disconnect-info, reason: 3
Aug 01 21:15:54 framework iwd[1747]: Received Deauthentication event, reason: 3, from_ap: false
Aug 01 21:15:54 framework systemd-logind[1748]: Lid opened.
Aug 01 21:15:54 framework kernel: random: crng reseeded on system resumption
Aug 01 21:15:54 framework kernel: Restarting tasks ... done.
Aug 01 21:15:54 framework kernel: OOM killer enabled.
Aug 01 21:15:54 framework kernel: [drm] ring sdma_32795.3.3 ib test pass
Aug 01 21:15:54 framework kernel: [drm] ring compute_32795.2.2 ib test pass
Aug 01 21:15:54 framework kernel: [drm] ring gfx_32795.1.1 ib test pass
Aug 01 21:15:54 framework kernel: [drm] ring sdma_32795.3.3 was added
Aug 01 21:15:54 framework kernel: [drm] ring compute_32795.2.2 was added
Aug 01 21:15:54 framework kernel: [drm] ring gfx_32795.1.1 was added
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring vpe uses VM inv eng 4 on hub 8
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring mes_kiq_3.1.0 uses VM inv eng 13 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring jpeg_dec_0 uses VM inv eng 1 on hub 8
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring vcn_unified_0 uses VM inv eng 0 on hub 8
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring sdma0 uses VM inv eng 12 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.3.1 uses VM inv eng 11 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.2.1 uses VM inv eng 10 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.1.1 uses VM inv eng 9 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.0.1 uses VM inv eng 8 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.3.0 uses VM inv eng 7 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.2.0 uses VM inv eng 6 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.1.0 uses VM inv eng 4 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring comp_1.0.0 uses VM inv eng 1 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: ring gfx_0.0.0 uses VM inv eng 0 on hub 0
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: SMU is resumed successfully!
Aug 01 21:15:54 framework kernel: nvme nvme0: 16/0/0 default/read/poll queues
Aug 01 21:15:54 framework kernel: amdgpu 0000:c1:00.0: amdgpu: SMU is resuming...
Aug 01 21:15:54 framework kernel: [drm] PCIE GART of 512M enabled (table at 0x0000008001700000).
Aug 01 21:15:54 framework kernel: nvme nvme0: D3 entry latency set to 10 seconds
Aug 01 21:15:54 framework kernel: ACPI: EC: interrupt unblocked
Aug 01 21:15:54 framework kernel: ACPI: EC: interrupt blocked
Aug 01 21:15:54 framework kernel: wlan0: deauthenticating from e6:55:b8:57:36:b1 by local choice (Reason: 3=DEAUTH_LEAVING)
Aug 01 21:15:54 framework kernel: printk: Suspending console(s) (use no_console_suspend to debug)
Aug 01 21:15:54 framework kernel: Freezing remaining freezable tasks completed (elapsed 0.000 seconds)
Aug 01 21:15:54 framework kernel: Freezing remaining freezable tasks
Aug 01 21:15:54 framework kernel: OOM killer disabled.
Aug 01 21:15:54 framework kernel: Freezing user space processes completed (elapsed 0.001 seconds)
Aug 01 21:15:54 framework kernel: Freezing user space processes
Aug 01 21:12:51 framework kernel: Filesystems sync: 0.014 seconds
Aug 01 21:12:51 framework kernel: PM: suspend entry (s2idle)
Aug 01 21:12:51 framework systemd-sleep[33661]: Performing sleep operation 'suspend'...
Aug 01 21:12:51 framework systemd-sleep[33661]: Successfully froze unit 'user.slice'.
Aug 01 21:12:51 framework systemd[1]: user.slice: Unit now frozen.
Aug 01 21:12:51 framework systemd[1]: session-1.scope: Unit now frozen-by-parent.
Aug 01 21:12:51 framework systemd[1]: user-1000.slice: Unit now frozen-by-parent.
Aug 01 21:12:51 framework systemd[1]: user@1000.service: Unit now frozen-by-parent.
Aug 01 21:12:51 framework systemd[1]: Starting System Suspend...
Aug 01 21:12:51 framework systemd[1]: Reached target Sleep.
Aug 01 21:12:51 framework systemd[1904]: Finished Omarchy Battery Monitor Check.
Aug 01 21:12:51 framework systemd[1904]: Starting Omarchy Battery Monitor Check...
Aug 01 21:12:51 framework systemd-logind[1748]: Suspending...
Aug 01 21:12:51 framework systemd-logind[1748]: Lid closed.
Aug 01 21:12:16 framework systemd[1904]: Finished Omarchy Battery Monitor Check.
Aug 01 21:12:16 framework systemd[1904]: Starting Omarchy Battery Monitor Check...
```
</details>

ituri commented:
Thank you for your assistance in debugging this further! No, the video is not from me.
I typed `date` shortly before closing the lid and right after opening the lid and launching a terminal window. 

Lid closed at 10:07:06, lid opened at 10:07:33.

```
Aug 02 10:07:53 tp systemd[873]: Starting Omarchy Battery Monitor Check...
Aug 02 10:07:31 tp blueman-applet[1800]: blueman-applet 10.07.31 WARNING  PluginManager:153 __load_plugin: Not loading PPPSupport because its conflict has higher priority
Aug 02 10:07:31 tp blueman-applet[1800]: blueman-applet 10.07.31 WARNING  PluginManager:153 __load_plugin: Not loading DhcpClient because its conflict has higher priority
Aug 02 10:07:31 tp blueman-applet[1800]: blueman-applet 10.07.31 WARNING  PluginManager:93 load_plugin: Failed to start plugin GameControllerWakelock: Only X11 platform is supported
Aug 02 10:07:31 tp nm-applet[1867]: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
Aug 02 10:07:31 tp blueberry-tray[1862]: gtk_widget_get_scale_factor: assertion 'GTK_IS_WIDGET (widget)' failed
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.081668 addonmanager.cpp:306] Unloading addon wayland
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.081559 addonmanager.cpp:306] Unloading addon imselector
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.077578 addonmanager.cpp:306] Unloading addon xcb
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.077189 addonmanager.cpp:306] Unloading addon quickphrase
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.077092 addonmanager.cpp:306] Unloading addon waylandim
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.076777 addonmanager.cpp:306] Unloading addon keyboard
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.076671 addonmanager.cpp:306] Unloading addon clipboard
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.076599 addonmanager.cpp:306] Unloading addon unicode
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.076501 addonmanager.cpp:200] Could not load addon dbus
Aug 02 10:07:31 tp fcitx5[1810]: E2025-08-02 10:07:31.076401 addonloader.cpp:96] Failed to create addon: dbus Unable to request dbus name. Is there another fcitx already running?
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.068694 addonmanager.cpp:204] Loaded addon unicode
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.068111 addonmanager.cpp:204] Loaded addon clipboard
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.063843 addonmanager.cpp:204] Loaded addon keyboard
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.031332 keyboard.cpp:226] Attempting to load keyboard from: ["/usr/share/X11/xkb"] Rule: evdev
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.020721 addonmanager.cpp:204] Loaded addon waylandim
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.020001 addonmanager.cpp:204] Loaded addon quickphrase
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.002188 addonmanager.cpp:204] Loaded addon xcb
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.002010 xcbconnection.cpp:204] X11 display: :2 is xwayland.
Aug 02 10:07:31 tp fcitx5[1810]: I2025-08-02 10:07:31.001857 xcbconnection.cpp:199] Connecting to X11 display, display name::2.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: Errors from xkbcomp are not fatal to the X server
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: > Warning:          Could not resolve keysym XF86DoNotDisturb
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: > Warning:          Could not resolve keysym XF86Accessibility
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: > Warning:          Could not resolve keysym XF86RefreshRateToggle
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: >                   X11 cannot support keycodes above 255.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: > Warning:          Unsupported maximum keycode 708, clipping.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1878]: The XKEYBOARD keymap compiler (xkbcomp) reports:
Aug 02 10:07:30 tp iwgtk[1807]: Icon theme 'breeze' is missing icon 'network-wireless-offline-symbolic': Overriding theme to Adwaita
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]: Autoready: Unit for hyprland.desktop is already active, restricting notification access.
Aug 02 10:07:30 tp systemd[873]: Started walker.
Aug 02 10:07:30 tp systemd[873]: Started swayosd-server.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1876]: Errors from xkbcomp are not fatal to the X server
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1876]: > Warning:          Could not resolve keysym XF86DoNotDisturb
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1876]: > Warning:          Could not resolve keysym XF86Accessibility
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1876]: > Warning:          Could not resolve keysym XF86RefreshRateToggle
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1876]: The XKEYBOARD keymap compiler (xkbcomp) reports:
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]:   HYPRLAND_INSTANCE_SIGNATURE
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]: Marking variables for later cleanup from systemd user manager on stop:
Aug 02 10:07:30 tp systemd[873]: Started xapp-sn-watcher.
Aug 02 10:07:30 tp systemd[873]: Finished Omarchy Battery Monitor Check.
Aug 02 10:07:30 tp systemd[873]: Started Print Queue Applet.
Aug 02 10:07:30 tp systemd[873]: Started NetworkManager Applet.
Aug 02 10:07:30 tp systemd[873]: Condition check resulted in User folders update being skipped.
Aug 02 10:07:30 tp systemd[873]: app-user\x2ddirs\x2dupdate\x2dgtk@autostart.service: Skipped due to 'exec-condition'.
Aug 02 10:07:30 tp systemd[873]: Condition check resulted in Plasma Session Restore being skipped.
Aug 02 10:07:30 tp systemd[873]: app-org.kde.plasma\x2dfallback\x2dsession\x2drestore@autostart.service: Skipped due to 'exec-condition'.
Aug 02 10:07:30 tp systemd[873]: Condition check resulted in Discover being skipped.
Aug 02 10:07:30 tp systemd[873]: app-org.kde.discover.notifier@autostart.service: Skipped due to 'exec-condition'.
Aug 02 10:07:30 tp systemd[873]: Condition check resulted in Secret Storage Service being skipped.
Aug 02 10:07:30 tp systemd[873]: app-gnome\x2dkeyring\x2dsecrets@autostart.service: Skipped due to 'exec-condition'.
Aug 02 10:07:30 tp systemd[873]: Started blueberry.
Aug 02 10:07:30 tp systemd[873]: Started Geoclue Demo agent.
Aug 02 10:07:30 tp systemd[873]: Started Bluetooth OBEX Agent.
Aug 02 10:07:30 tp systemd[873]: Condition check resulted in Certificate and Key Storage being skipped.
Aug 02 10:07:30 tp systemd[873]: app-gnome\x2dkeyring\x2dpkcs11@autostart.service: Skipped due to 'exec-condition'.
Aug 02 10:07:30 tp systemd[873]: Condition check resulted in AT-SPI D-Bus Bus being skipped.
Aug 02 10:07:30 tp systemd[873]: app-at\x2dspi\x2ddbus\x2dbus@autostart.service: Skipped due to 'exec-condition'.
Aug 02 10:07:30 tp systemd[873]: Started Blueman Applet.
Aug 02 10:07:30 tp iwgtk[1807]: Unknown key gtk-modules in /home/phil/.config/gtk-4.0/settings.ini
Aug 02 10:07:30 tp systemd[873]: Started Fcitx 5.
Aug 02 10:07:30 tp systemd[873]: Started iwgtk.
Aug 02 10:07:30 tp fcitx5[1810]: I2025-08-02 10:07:30.786808 addonmanager.cpp:204] Loaded addon imselector
Aug 02 10:07:30 tp fcitx5[1810]: I2025-08-02 10:07:30.786278 addonmanager.cpp:204] Loaded addon wayland
Aug 02 10:07:30 tp fcitx5[1810]: I2025-08-02 10:07:30.776569 instance.cpp:1401] Override Disabled Addons: {}
Aug 02 10:07:30 tp fcitx5[1810]: I2025-08-02 10:07:30.762481 instance.cpp:1400] Override Enabled Addons: {}
Aug 02 10:07:30 tp systemd[873]: Starting Omarchy Battery Monitor Check...
Aug 02 10:07:30 tp systemd[873]: Starting xapp-sn-watcher...
Aug 02 10:07:30 tp systemd[873]: Starting User folders update...
Aug 02 10:07:30 tp systemd[873]: Starting Print Queue Applet...
Aug 02 10:07:30 tp systemd[873]: Starting Plasma Session Restore...
Aug 02 10:07:30 tp systemd[873]: Starting Discover...
Aug 02 10:07:30 tp systemd[873]: Starting Fcitx 5...
Aug 02 10:07:30 tp systemd[873]: Starting NetworkManager Applet...
Aug 02 10:07:30 tp systemd[873]: Starting iwgtk...
Aug 02 10:07:30 tp systemd[873]: Starting Secret Storage Service...
Aug 02 10:07:30 tp systemd[873]: Starting Certificate and Key Storage...
Aug 02 10:07:30 tp systemd[873]: Starting Geoclue Demo agent...
Aug 02 10:07:30 tp systemd[873]: Starting Blueman Applet...
Aug 02 10:07:30 tp systemd[873]: Starting blueberry...
Aug 02 10:07:30 tp systemd[873]: Starting Bluetooth OBEX Agent...
Aug 02 10:07:30 tp systemd[873]: Starting AT-SPI D-Bus Bus...
Aug 02 10:07:30 tp systemd[873]: Reached target Startup of XDG autostart applications.
Aug 02 10:07:30 tp systemd[873]: Reached target XDG Autostart for session of hyprland.desktop Wayland compositor.
Aug 02 10:07:30 tp systemd[873]: Reached target Current graphical user session.
Aug 02 10:07:30 tp systemd[873]: Reached target Session of hyprland.desktop Wayland compositor.
Aug 02 10:07:30 tp systemd[873]: Started waybar.
Aug 02 10:07:30 tp systemd[873]: Started swaybg.
Aug 02 10:07:30 tp systemd[873]: Started mako.
Aug 02 10:07:30 tp systemd[873]: Started hypridle.
Aug 02 10:07:30 tp systemd[873]: Started fcitx5.
Aug 02 10:07:30 tp systemd[873]: Finished Wait for WAYLAND_DISPLAY and other variables.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]:   WAYLAND_DISPLAY (already set)
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]:   HYPRLAND_INSTANCE_SIGNATURE (already set)
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]:   DISPLAY (already set)
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]: Exporting variables to systemd user manager:
Aug 02 10:07:30 tp systemd[873]: Started Main service for Hyprland, An intelligent dynamic tiling Wayland compositor.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1738]: All expected variables appeared in activation environment.
Aug 02 10:07:30 tp uwsm_waitenv[1735]: All expected variables appeared in activation environment.
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1736]: Fontconfig warning: using without calling FcInit()
Aug 02 10:07:30 tp uwsm_hyprland.desktop[1736]: unable to lock lockfile /run/user/1000/wayland-1.lock, maybe another compositor is running
Aug 02 10:07:29 tp uwsm[1710]: waitpid: Holding login session 5 open until PID 1736 exits
Aug 02 10:07:29 tp seamless-login[1710]: waitpid: Holding login session 5 open until PID 1736 exits
Aug 02 10:07:29 tp uwsm_hyprland.desktop[1738]:   WAYLAND_DISPLAY
Aug 02 10:07:29 tp uwsm_hyprland.desktop[1738]: Still expecting:
Aug 02 10:07:29 tp uwsm_hyprland.desktop[1738]:   HYPRLAND_INSTANCE_SIGNATURE
Aug 02 10:07:29 tp uwsm_hyprland.desktop[1738]: Expected variables appeared in activation environment:
Aug 02 10:07:29 tp uwsm_waitenv[1735]:   WAYLAND_DISPLAY
Aug 02 10:07:29 tp uwsm_waitenv[1735]: Still expecting:
Aug 02 10:07:29 tp uwsm_waitenv[1735]:   HYPRLAND_INSTANCE_SIGNATURE
Aug 02 10:07:29 tp uwsm_waitenv[1735]: Expected variables appeared in activation environment:
Aug 02 10:07:29 tp uwsm_hyprland.desktop[1736]: Starting: Hyprland...
Aug 02 10:07:29 tp systemd-timesyncd[389]: Network configuration changed, trying to establish connection.
Aug 02 10:07:29 tp systemd[873]: Starting Main service for Hyprland, An intelligent dynamic tiling Wayland compositor...
Aug 02 10:07:29 tp systemd[873]: Starting Wait for WAYLAND_DISPLAY and other variables...
Aug 02 10:07:29 tp systemd[873]: Reached target Session services which should run early before the graphical session is brought up.
Aug 02 10:07:29 tp systemd[873]: Reached target Preparation for session of hyprland.desktop Wayland compositor.
Aug 02 10:07:29 tp systemd[873]: Finished Environment preloader for Hyprland, An intelligent dynamic tiling Wayland compositor.
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   WAYLAND_DISPLAY
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   DISPLAY
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Removing variables from systemd user manager:
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_VTNR
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_STATE_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_TYPE
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_ID
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_DESKTOP
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_CLASS
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SEAT
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_MENU_PREFIX
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_DATA_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_DATA_DIRS
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CURRENT_DESKTOP (already set)
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CONFIG_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CONFIG_DIRS
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CACHE_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_BACKEND
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   UWSM_WAIT_VARNAMES
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   UWSM_FINALIZE_VARNAMES
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   SYSTEMD_EXEC_PID
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   PATH (updating)
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   MEMORY_PRESSURE_WRITE
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   MEMORY_PRESSURE_WATCH
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   MAIL
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   JOURNAL_STREAM
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Exporting variables to systemd user manager:
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_VTNR
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_STATE_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_TYPE
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_ID
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_DESKTOP
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SESSION_CLASS
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_SEAT
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_MENU_PREFIX
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_DATA_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_DATA_DIRS
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CURRENT_DESKTOP
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CONFIG_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CONFIG_DIRS
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_CACHE_HOME
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   XDG_BACKEND
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   UWSM_WAIT_VARNAMES
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   UWSM_FINALIZE_VARNAMES
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   SYSTEMD_EXEC_PID
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   PATH
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   MEMORY_PRESSURE_WRITE
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   MEMORY_PRESSURE_WATCH
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   MAIL
Aug 02 10:07:29 tp uwsm_env-preloader[1711]:   JOURNAL_STREAM
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Marking variables for later cleanup from systemd user manager on stop:
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Applying quirks for "hyprland".
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Loading plugin "/usr/share/uwsm/plugins/hyprland.sh".
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Got saved login session variables.
Aug 02 10:07:29 tp uwsm_env-preloader[1711]: Preparing environment for Hyprland...
Aug 02 10:07:29 tp systemd[873]: Starting Environment preloader for Hyprland, An intelligent dynamic tiling Wayland compositor...
Aug 02 10:07:29 tp systemd[873]: Started Bind graphical session to PID 1675.
Aug 02 10:07:29 tp systemd[873]: Starting Bind graphical session to PID 1675...
Aug 02 10:07:29 tp systemd[873]: Created slice User Background Tasks Slice.
Aug 02 10:07:29 tp uwsm[1675]: Starting hyprland.desktop and waiting while it is running...
Aug 02 10:07:29 tp seamless-login[1675]: Starting hyprland.desktop and waiting while it is running...
Aug 02 10:07:29 tp systemd[873]: Reloading finished in 270 ms.
Aug 02 10:07:28 tp systemd[873]: Reloading...
Aug 02 10:07:28 tp systemd[873]: Reload requested from client PID 1675 ('uwsm')...
Aug 02 10:07:28 tp uwsm[1675]: Reloading systemd user manager.
Aug 02 10:07:28 tp seamless-login[1675]: Reloading systemd user manager.
Aug 02 10:07:28 tp seamless-login[1675]: Created "plasma-xdg-desktop-portal-kde.service.d/order-tweak.conf" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created "plasma-xdg-desktop-portal-kde.service.d/order-tweak.conf" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created unit subdir "plasma-xdg-desktop-portal-kde.service.d/" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created unit subdir "plasma-xdg-desktop-portal-kde.service.d/" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created "app-flatpak-.scope.d/order-tweak.conf" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created "app-flatpak-.scope.d/order-tweak.conf" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created unit subdir "app-flatpak-.scope.d/" (run).
ug 02 10:07:28 tp seamless-login[1675]: Created unit subdir "app-flatpak-.scope.d/" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created "app-@autostart.service.d/slice-tweak.conf" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created "app-@autostart.service.d/slice-tweak.conf" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created unit subdir "app-@autostart.service.d/" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created unit subdir "app-@autostart.service.d/" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created "wayland-wm@hyprland.desktop.service.d/50_custom.conf" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created "wayland-wm@hyprland.desktop.service.d/50_custom.conf" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created unit subdir "wayland-wm@hyprland.desktop.service.d/" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created unit subdir "wayland-wm@hyprland.desktop.service.d/" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created "wayland-wm-env@hyprland.desktop.service.d/50_custom.conf" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created "wayland-wm-env@hyprland.desktop.service.d/50_custom.conf" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created unit subdir "wayland-wm-env@hyprland.desktop.service.d/" (run).
Aug 02 10:07:28 tp uwsm[1675]: Created dir "/run/user/1000/systemd/user/"
Aug 02 10:07:28 tp seamless-login[1675]: Created unit subdir "wayland-wm-env@hyprland.desktop.service.d/" (run).
Aug 02 10:07:28 tp seamless-login[1675]: Created dir "/run/user/1000/systemd/user/"
Aug 02 10:07:28 tp uwsm[1675]: Selected compositor ID: hyprland.desktop
                                         Command Line: Hyprland
                                     Plugin/binary ID: hyprland
                                Initial Desktop Names: Hyprland
                                                 Name: Hyprland
                                          Description: An intelligent dynamic tiling Wayland compositor
Aug 02 10:07:28 tp seamless-login[1675]:            Description: An intelligent dynamic tiling Wayland compositor
Aug 02 10:07:28 tp seamless-login[1675]:                   Name: Hyprland
Aug 02 10:07:28 tp seamless-login[1675]:  Initial Desktop Names: Hyprland
Aug 02 10:07:28 tp seamless-login[1675]:       Plugin/binary ID: hyprland
Aug 02 10:07:28 tp seamless-login[1675]:           Command Line: Hyprland
Aug 02 10:07:28 tp seamless-login[1675]: Selected compositor ID: hyprland.desktop
Aug 02 10:07:28 tp systemd[1]: Started Session 5 of User phil.
Aug 02 10:07:28 tp systemd-logind[631]: New session 5 of user phil.
Aug 02 10:07:28 tp (ss-login)[1675]: pam_unix(login:session): session opened for user phil(uid=1000) by phil(uid=0)
Aug 02 10:07:28 tp systemd[1]: Started Omarchy Seamless Auto-Login.
Aug 02 10:07:28 tp systemd[1]: omarchy-seamless-login.service: Scheduled restart job, restart counter is at 1.
Aug 02 10:07:27 tp systemd-timesyncd[389]: Network configuration changed, trying to establish connection.
Aug 02 10:07:27 tp systemd-timesyncd[389]: Network configuration changed, trying to establish connection.
Aug 02 10:07:26 tp systemd-timesyncd[389]: Network configuration changed, trying to establish connection.
Aug 02 10:07:26 tp systemd-timesyncd[389]: Network configuration changed, trying to establish connection.
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: DHCPv4 address 192.168.1.26/23, gateway 192.168.1.1 acquired from 192.168.1.1
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: Gained carrier
Aug 02 10:07:26 tp iwd[628]: event: state, old: connecting (auto), new: connected
Aug 02 10:07:26 tp systemd-resolved[388]: Switching to fallback DNS server 1.1.1.1#cloudflare-dns.com.
Aug 02 10:07:26 tp kernel: wlan0: associated
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: Connected WiFi access point: sysv (1a:e8:29:41:a2:a7)
Aug 02 10:07:26 tp kernel: wlan0: RX AssocResp from 1a:e8:29:41:a2:a7 (capab=0x1111 status=0 aid=10)
Aug 02 10:07:26 tp kernel: wlan0: associate with 1a:e8:29:41:a2:a7 (try 1/3)
Aug 02 10:07:26 tp kernel: wlan0: authenticated
Aug 02 10:07:26 tp kernel: wlan0: send auth to 1a:e8:29:41:a2:a7 (try 1/3)
Aug 02 10:07:26 tp kernel: wlan0: authenticate with 1a:e8:29:41:a2:a7 (local address=a0:e7:0b:24:43:39)
Aug 02 10:07:26 tp iwd[628]: event: state, old: autoconnect_quick, new: connecting (auto)
Aug 02 10:07:26 tp iwd[628]: event: connect-info, ssid: sysv, bss: 1a:e8:29:41:a2:a7, signal: -60, load: 4/255
Aug 02 10:07:26 tp systemd[1]: Started Fingerprint Authentication Daemon.
Aug 02 10:07:26 tp fprintd[1662]: Device responded with error: 789 retry: 1
Aug 02 10:07:26 tp systemd[1]: Starting Fingerprint Authentication Daemon...
Aug 02 10:07:26 tp systemd-networkd[411]: enp0s31f6: Reconfiguring with /etc/systemd/network/20-ethernet.network.
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: Reconfiguring with /etc/systemd/network/20-wlan.network.
Aug 02 10:07:26 tp systemd-resolved[388]: Resetting learnt feature levels on all servers.
Aug 02 10:07:26 tp systemd-resolved[388]: Closing all remaining TCP connections.
Aug 02 10:07:26 tp systemd-logind[631]: Operation 'suspend' finished.
Aug 02 10:07:26 tp systemd[1]: Stopped target Suspend.
Aug 02 10:07:26 tp systemd-logind[631]: Removed session 1.
Aug 02 10:07:26 tp systemd[1]: Reached target Suspend.
Aug 02 10:07:26 tp systemd[1]: Stopped target Sleep.
Aug 02 10:07:26 tp systemd[1]: session-1.scope: Deactivated successfully.
Aug 02 10:07:26 tp systemd-logind[631]: Session 1 logged out. Waiting for processes to exit.
Aug 02 10:07:26 tp systemd[1]: Finished System Suspend.
Aug 02 10:07:26 tp systemd[1]: systemd-suspend.service: Deactivated successfully.
Aug 02 10:07:26 tp systemd[1]: omarchy-seamless-login.service: Failed with result 'exit-code'.
Aug 02 10:07:26 tp systemd[1]: omarchy-seamless-login.service: Main process exited, code=exited, status=1/FAILURE
Aug 02 10:07:26 tp systemd[1]: user@1000.service: Unit now thawed.
Aug 02 10:07:26 tp systemd[1]: session-4.scope: Unit now thawed.
Aug 02 10:07:26 tp systemd[1]: session-1.scope: Unit now thawed.
Aug 02 10:07:26 tp systemd[1]: user-1000.slice: Unit now thawed.
Aug 02 10:07:26 tp systemd-sleep[1595]: Successfully thawed unit 'user.slice'.
Aug 02 10:07:26 tp systemd[1]: user.slice: Unit now thawed.
Aug 02 10:07:26 tp seamless-login[834]: VT_WAITACTIVE failed: Interrupted system call
Aug 02 10:07:26 tp kernel: PM: suspend exit
Aug 02 10:07:26 tp bluetoothd[624]: Controller resume with wake event 0x0
Aug 02 10:07:26 tp systemd-sleep[1595]: System returned from sleep operation 'suspend'.
Aug 02 10:07:26 tp systemd-resolved[388]: Clock change detected. Flushing caches.
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: DHCP lease lost
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: Lost carrier
Aug 02 10:07:26 tp systemd-logind[631]: Lid opened.
Aug 02 10:07:26 tp iwd[628]: event: state, old: disconnected, new: autoconnect_quick
Aug 02 10:07:26 tp iwd[628]: event: state, old: connected, new: disconnected
Aug 02 10:07:26 tp iwd[628]: event: disconnect-info, reason: 3
Aug 02 10:07:26 tp iwd[628]: Received Deauthentication event, reason: 3, from_ap: false
Aug 02 10:07:26 tp kernel: mei_pxp 0000:00:16.0-fbf6fcf1-96cf-4e2e-a6a6-1bab8cbe36b1: bound 0000:00:02.0 (ops i915_pxp_tee_component_ops [i915])
Aug 02 10:07:26 tp kernel: random: crng reseeded on system resumption
Aug 02 10:07:26 tp kernel: done.
Aug 02 10:07:26 tp kernel: mei_hdcp 0000:00:16.0-b638ab7e-94e2-4ea2-a552-d1c54b627f04: bound 0000:00:02.0 (ops i915_hdcp_ops [i915])
Aug 02 10:07:26 tp kernel: Restarting tasks ...
Aug 02 10:07:26 tp kernel: OOM killer enabled.
Aug 02 10:07:26 tp kernel: nvme nvme0: 8/0/0 default/read/poll queues
Aug 02 10:07:26 tp kernel: ACPI: EC: interrupt unblocked
Aug 02 10:07:26 tp kernel: ACPI: EC: interrupt blocked
Aug 02 10:07:26 tp kernel: e1000e: EEE TX LPI TIMER: 00000011
Aug 02 10:07:26 tp kernel: wlan0: deauthenticating from 1a:e8:29:41:a2:a7 by local choice (Reason: 3=DEAUTH_LEAVING)
Aug 02 10:07:26 tp kernel: printk: Suspending console(s) (use no_console_suspend to debug)
Aug 02 10:07:26 tp kernel: Freezing remaining freezable tasks completed (elapsed 0.000 seconds)
Aug 02 10:07:26 tp kernel: Freezing remaining freezable tasks
Aug 02 10:07:26 tp kernel: OOM killer disabled.
Aug 02 10:07:26 tp kernel: Freezing user space processes completed (elapsed 0.001 seconds)
Aug 02 10:07:18 tp kernel: Freezing user space processes
Aug 02 10:07:18 tp kernel: Filesystems sync: 0.065 seconds
Aug 02 10:07:17 tp kernel: PM: suspend entry (s2idle)
Aug 02 10:07:17 tp systemd-sleep[1595]: Performing sleep operation 'suspend'...
Aug 02 10:07:17 tp systemd-sleep[1595]: Successfully froze unit 'user.slice'.
Aug 02 10:07:17 tp systemd[1]: user-1000.slice: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: user.slice: Unit now frozen.
Aug 02 10:07:17 tp systemd[1]: session-1.scope: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: session-4.scope: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: user@1000.service: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: Starting System Suspend...
Aug 02 10:07:17 tp systemd[1]: Reached target Sleep.
Aug 02 10:07:17 tp polkitd[650]: Unregistered Authentication Agent for unix-process:1573:3025 (system bus name :1.53, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8) (disconnected from bus)
Aug 02 10:07:17 tp polkitd[650]: Registered Authentication Agent for unix-process:1573:3025 (system bus name :1.53 [/usr/bin/pkttyagent --notify-fd 7 --fallback], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8)
Aug 02 10:07:17 tp systemd-logind[631]: Suspending...
Aug 02 10:07:09 tp systemd-logind[631]: Lid closed.
Aug 02 10:07:06 tp syncthing[757]: [Q47IP] INFO: Lost primary connection to 2HM6PSG at 192.168.1.26:22000-192.168.1.118:22000/quic-server/TLS1.3-TLS_AES_128_GCM_SHA256/LAN-P20-62NSIN3N9NGVMUTINR4TFJ02AU: replacing connection (1 remain)
Aug 02 10:07:06 tp syncthing[757]: [Q47IP] INFO: Additional connection (+1) for device 2HM6PSG at 192.168.1.26:22000-192.168.1.118:22000/tcp-client/TLS1.3-TLS_AES_128_GCM_SHA256/LAN-P10-62NSIOA0SBM4UPLKGJTD1CJ83O
Aug 02 10:07:06 tp syncthing[757]: [Q47IP] INFO: Established secure connection to 2HM6PSG at 192.168.1.26:22000-192.168.1.118:22000/tcp-client/TLS1.3-TLS_AES_128_GCM_SHA256/LAN-P10-62NSIOA0SBM4UPLKGJTD1CJ83O
Aug 02 10:07:06 tp systemd[1]: user-968.slice: Consumed 543ms CPU time, 257.2M memory peak.
Aug 02 10:07:06 tp systemd[1]: Removed slice User Slice of UID 968.
Aug 02 10:07:06 tp systemd[1]: Stopped User Runtime Directory /run/user/968.
Aug 02 10:07:06 tp systemd[1]: user-runtime-dir@968.service: Deactivated successfully.
Aug 02 10:07:06 tp systemd[1]: run-user-968.mount: Deactivated successfully.
Aug 02 10:07:06 tp systemd[1]: Stopping User Runtime Directory /run/user/968...
Aug 02 10:07:06 tp systemd[1]: Stopped User Manager for UID 968.
Aug 02 10:07:06 tp systemd[1]: user@968.service: Deactivated successfully.
Aug 02 10:07:06 tp systemd-logind[631]: Removed session 3.
Aug 02 10:07:06 tp systemd[1178]: Reached target Exit the Session.
Aug 02 10:07:26 tp systemd[1]: omarchy-seamless-login.service: Failed with result 'exit-code'.
Aug 02 10:07:26 tp systemd[1]: omarchy-seamless-login.service: Main process exited, code=exited, status=1/FAILURE
Aug 02 10:07:26 tp systemd[1]: user@1000.service: Unit now thawed.
Aug 02 10:07:26 tp systemd[1]: session-4.scope: Unit now thawed.
Aug 02 10:07:26 tp systemd[1]: session-1.scope: Unit now thawed.
Aug 02 10:07:26 tp systemd[1]: user-1000.slice: Unit now thawed.
Aug 02 10:07:26 tp systemd-sleep[1595]: Successfully thawed unit 'user.slice'.
Aug 02 10:07:26 tp systemd[1]: user.slice: Unit now thawed.
Aug 02 10:07:26 tp seamless-login[834]: VT_WAITACTIVE failed: Interrupted system call
Aug 02 10:07:26 tp kernel: PM: suspend exit
Aug 02 10:07:26 tp bluetoothd[624]: Controller resume with wake event 0x0
Aug 02 10:07:26 tp systemd-sleep[1595]: System returned from sleep operation 'suspend'.
Aug 02 10:07:26 tp systemd-resolved[388]: Clock change detected. Flushing caches.
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: DHCP lease lost
Aug 02 10:07:26 tp systemd-networkd[411]: wlan0: Lost carrier
Aug 02 10:07:26 tp systemd-logind[631]: Lid opened.
Aug 02 10:07:26 tp iwd[628]: event: state, old: disconnected, new: autoconnect_quick
Aug 02 10:07:26 tp iwd[628]: event: state, old: connected, new: disconnected
Aug 02 10:07:26 tp iwd[628]: event: disconnect-info, reason: 3
Aug 02 10:07:26 tp iwd[628]: Received Deauthentication event, reason: 3, from_ap: false
Aug 02 10:07:26 tp kernel: mei_pxp 0000:00:16.0-fbf6fcf1-96cf-4e2e-a6a6-1bab8cbe36b1: bound 0000:00:02.0 (ops i915_pxp_tee_component_ops [i915])
Aug 02 10:07:26 tp kernel: random: crng reseeded on system resumption
Aug 02 10:07:26 tp kernel: done.
Aug 02 10:07:26 tp kernel: mei_hdcp 0000:00:16.0-b638ab7e-94e2-4ea2-a552-d1c54b627f04: bound 0000:00:02.0 (ops i915_hdcp_ops [i915])
Aug 02 10:07:26 tp kernel: Restarting tasks ...
Aug 02 10:07:26 tp kernel: OOM killer enabled.
Aug 02 10:07:26 tp kernel: nvme nvme0: 8/0/0 default/read/poll queues
Aug 02 10:07:26 tp kernel: ACPI: EC: interrupt unblocked
Aug 02 10:07:26 tp kernel: ACPI: EC: interrupt blocked
Aug 02 10:07:26 tp kernel: e1000e: EEE TX LPI TIMER: 00000011
Aug 02 10:07:26 tp kernel: wlan0: deauthenticating from 1a:e8:29:41:a2:a7 by local choice (Reason: 3=DEAUTH_LEAVING)
Aug 02 10:07:26 tp kernel: printk: Suspending console(s) (use no_console_suspend to debug)
Aug 02 10:07:26 tp kernel: Freezing remaining freezable tasks completed (elapsed 0.000 seconds)
Aug 02 10:07:26 tp kernel: Freezing remaining freezable tasks
Aug 02 10:07:26 tp kernel: OOM killer disabled.
Aug 02 10:07:26 tp kernel: Freezing user space processes completed (elapsed 0.001 seconds)
Aug 02 10:07:18 tp kernel: Freezing user space processes
Aug 02 10:07:18 tp kernel: Filesystems sync: 0.065 seconds
Aug 02 10:07:17 tp kernel: PM: suspend entry (s2idle)
Aug 02 10:07:17 tp systemd-sleep[1595]: Performing sleep operation 'suspend'...
Aug 02 10:07:17 tp systemd-sleep[1595]: Successfully froze unit 'user.slice'.
Aug 02 10:07:17 tp systemd[1]: user-1000.slice: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: user.slice: Unit now frozen.
Aug 02 10:07:17 tp systemd[1]: session-1.scope: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: session-4.scope: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: user@1000.service: Unit now frozen-by-parent.
Aug 02 10:07:17 tp systemd[1]: Starting System Suspend...
Aug 02 10:07:17 tp systemd[1]: Reached target Sleep.
Aug 02 10:07:17 tp polkitd[650]: Unregistered Authentication Agent for unix-process:1573:3025 (system bus name :1.53, object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8) (disconnected from bus)
Aug 02 10:07:17 tp polkitd[650]: Registered Authentication Agent for unix-process:1573:3025 (system bus name :1.53 [/usr/bin/pkttyagent --notify-fd 7 --fallback], object path /org/freedesktop/PolicyKit1/AuthenticationAgent, locale en_US.UTF-8)
Aug 02 10:07:17 tp systemd-logind[631]: Suspending...
Aug 02 10:07:09 tp systemd-logind[631]: Lid closed.
Aug 02 10:07:06 tp syncthing[757]: [Q47IP] INFO: Lost primary connection to 2HM6PSG at 192.168.1.26:22000-192.168.1.118:22000/quic-server/TLS1.3-TLS_AES_128_GCM_SHA256/LAN-P20-62NSIN3N9NGVMUTINR4TFJ02AU: replacing connection (1 remain)
Aug 02 10:07:06 tp syncthing[757]: [Q47IP] INFO: Additional connection (+1) for device 2HM6PSG at 192.168.1.26:22000-192.168.1.118:22000/tcp-client/TLS1.3-TLS_AES_128_GCM_SHA256/LAN-P10-62NSIOA0SBM4UPLKGJTD1CJ83O
Aug 02 10:07:06 tp syncthing[757]: [Q47IP] INFO: Established secure connection to 2HM6PSG at 192.168.1.26:22000-192.168.1.118:22000/tcp-client/TLS1.3-TLS_AES_128_GCM_SHA256/LAN-P10-62NSIOA0SBM4UPLKGJTD1CJ83O
Aug 02 10:07:06 tp systemd[1]: user-968.slice: Consumed 543ms CPU time, 257.2M memory peak.
Aug 02 10:07:06 tp systemd[1]: Removed slice User Slice of UID 968.
Aug 02 10:07:06 tp systemd[1]: Stopped User Runtime Directory /run/user/968.
Aug 02 10:07:06 tp systemd[1]: user-runtime-dir@968.service: Deactivated successfully.
Aug 02 10:07:06 tp systemd[1]: run-user-968.mount: Deactivated successfully.
Aug 02 10:07:06 tp systemd[1]: Stopping User Runtime Directory /run/user/968...
Aug 02 10:07:06 tp systemd[1]: Stopped User Manager for UID 968.
Aug 02 10:07:06 tp systemd[1]: user@968.service: Deactivated successfully.
Aug 02 10:07:06 tp systemd-logind[631]: Removed session 3.
Aug 02 10:07:06 tp systemd[1178]: Reached target Exit the Session.
Aug 02 10:07:06 tp systemd[1178]: Finished Exit the Session.
Aug 02 10:07:06 tp systemd[1178]: Reached target Shutdown.
Aug 02 10:07:06 tp systemd[1178]: Removed slice User Application Slice.
Aug 02 10:07:06 tp systemd[1178]: Closed D-Bus User Message Bus Socket.
Aug 02 10:07:06 tp systemd[1178]: Removed slice User Core Session Slice.
Aug 02 10:07:06 tp systemd[1178]: Stopped D-Bus User Message Bus.
Aug 02 10:07:06 tp systemd[1178]: Stopping D-Bus User Message Bus...
Aug 02 10:07:06 tp dbus-broker[1193]: Dispatched 201 messages @ 1(±2)μs / message.
Aug 02 10:07:06 tp systemd[1178]: Closed PipeWire Multimedia System Sockets.
Aug 02 10:07:06 tp systemd[1178]: Closed PipeWire PulseAudio.
Aug 02 10:07:06 tp systemd[1178]: Closed p11-kit server.
Aug 02 10:07:06 tp systemd[1178]: Closed GnuPG public key management service.
Aug 02 10:07:06 tp systemd[1178]: Closed GnuPG cryptographic agent and passphrase cache.
Aug 02 10:07:06 tp systemd[1178]: Closed GnuPG cryptographic agent (ssh-agent emulation).
Aug 02 10:07:06 tp systemd[1178]: Closed GnuPG cryptographic agent and passphrase cache (restricted).
Aug 02 10:07:06 tp systemd[1178]: Closed GnuPG cryptographic agent and passphrase cache (access for web browsers).
Aug 02 10:07:06 tp systemd[1178]: Closed GNOME Keyring daemon.
Aug 02 10:07:06 tp systemd[1178]: Closed GnuPG network certificate management daemon.
Aug 02 10:07:06 tp systemd[1178]: Stopped target Timers.
Aug 02 10:07:06 tp systemd[1178]: Stopped target Sockets.
Aug 02 10:07:06 tp systemd[1178]: Stopped Submitting pending crash events (file monitor).
Aug 02 10:07:06 tp systemd[1178]: Stopped target Paths.
Aug 02 10:07:06 tp systemd[1178]: Stopped target Basic System.
Aug 02 10:07:06 tp systemd[1178]: Stopped target Main User Target.
Aug 02 10:07:06 tp systemd[1178]: Activating special unit Exit the Session...
Aug 02 10:07:06 tp systemd[1]: Stopping User Manager for UID 968...
Aug 02 10:06:59 tp systemd[1]: Started Power Profiles daemon.
Aug 02 10:06:59 tp systemd[1]: Starting Power Profiles daemon...
Aug 02 10:06:59 tp systemd[873]: Started Portal service.
```

ituri commented:
Aaaaand I've just found out that, unfortunately, the crashes also continue to happen with `Restart: always`. For the log above, I had set back everything back to default. After that, I returned to `RestartSec: 5`. However, it seems that this is not enough. Only after setting `Restart: on-failure` was I able to reopen the lid without any crashes.

ituri commented:
Starting with v1.10.0, my fix doesn't work any longer. Waking up from sleep crashes hyprland and all my open windows are lost and walker is not working any more.

ryanrhughes commented:
Found actual issue due to having seamless-login and SDDM in #412.
